challenges/functions.js

```diff
@@ -7,25 +7,34 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
+const consume = (arg1, arg2, cb) => console.log(cb(arg1, arg2));
+
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
 
+const add = (x, y) => x + y;
+const multiply = (x, y) => x * y;
+const greeting = (firstName, lastName) => {
+  return `Hello ${firstName} ${lastName}, nice to meet you!`;
+};
```

Comment:
since you did the rest in arrow funcs I'm assuming you didn't know that this will work as well `const greeting = (firstName, lastName) => `Hello ${firstName} ${lastName}, nice to meet you!`;`